packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -0,0 +1,313 @@
+import Check from "./Check.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+import Resource from "./Resource.js";
+import RuntimeError from "./RuntimeError.js";
+
+/**
+ * @typedef {Object} GeometryOptions
+ * @property {boolean} includeLines
+ * @property {number} chordTol
+ * @property {number} angleTol
+ * @property {number} decimationTol
+ * @property {number} maxEdgeLength
+ * @property {number} minBRepFeatureSize
+ * @property {number} minLineStyleComponentSize
+ */
+
+/**
+ * @typedef {Object} ViewDefinitionFilter
+ * @property {string[]} models Array of included model IDs.
+ * @property {string[]} categories Array of included category IDs.
+ * @property {string[]} neverDrawn Array of element IDs to filter out.
```

Comment:
You can remove these options.
We will remove it from the documentation in the next days, so using it may break your sample